RadixWallet/Core/FeaturePrelude/AddressView/AddressDetails+View.swift

```diff
@@ -94,22 +94,35 @@ public extension AddressDetails {
 			.animation(.easeInOut, value: store.qrImage.isSuccess)
 		}
 
+		@ViewBuilder
 		private var actions: some SwiftUI.View {
-			HStack(spacing: .large3) {
-				Button(L10n.AddressDetails.copy, image: .copy) {
+			FlowLayout(multilineAlignment: .center, spacing: .large3) {
```

Comment:
I believe we should decrease this spacing as well.

Reply:
Hmm, not sure how much. Reducing this space will deviate from the design.